### PR TITLE
Amber workspace

### PR DESCRIPTION
**Pull request by Msambere**
Hey ladies I wrote some tests for our routes and made some changes (standard responses, name convention, etc). Take a look and let me know what you think!

### CODE REVIEW

File: app/models/board.py [hunk 1]
```diff
@@ -5,14 +5,20 @@ class Board(db.Model):
...
-        for card in self.cards:
-            cards.append(card.to_dict())
-            result["cards"] = cards
+        # cards = []
+        # for card in self.cards:
+        #     cards.append(card.to_dict())
+        result = {
+            "id": self.id, 
+            "title": self.title, 
+            "cards": len(self.cards),
```
Comment (SalmaAnany) [inline app/models/board.py]: why? this going to make multiple calls

File: app/models/card.py [hunk 2]
```diff
@@ -6,14 +6,17 @@ class Card(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     text: Mapped[str] = mapped_column(nullable=False)
     likes: Mapped[int] = mapped_column(default=0)
-    board_id = db.Column(Integer, ForeignKey('board.id'), nullable=True)
-    board = relationship('Board', back_populates='cards')
+    # board_id = db.Column(Integer, ForeignKey('board.id'), nullable=True)
+    # board = relationship('Board', back_populates='cards')
+    board_id: Mapped[int] = mapped_column(ForeignKey("board.id"))
+    board: Mapped["Board"] = relationship(back_populates="cards")
```
Comment (SalmaAnany) [inline app/models/card.py:12]: Why is this even different?

File: app/routes/board_route.py [hunk 3]
```diff
@@ -1,67 +1,78 @@
...
+bp = Blueprint("board_bp", __name__,url_prefix="/boards")
 
 
-@board_bp.post("")
+@bp.post("")
 def create_board():
     response_data = request.get_json()
-    if not response_data.get("title"):
-        abort(make_response({"details": "Invalid data"}, 400))
+    # if not response_data.get("title"):
```
Comment (SalmaAnany) [inline app/routes/board_route.py:13]: Is there a reason to remove it?

File: app/models/board.py (hunk 1 above)
Comment (Msambere) [reply app/models/board.py]: Oh, I changed this back! I just made it shorter when I was testing the API. I think I forgot to push that change.

File: app/models/card.py (hunk 2 above)
Comment (Msambere) [reply app/models/card.py:12]: Line 9: The "db.Column" syntax is becoming outdated and when I was working on a previous API Kelsey told me we should use the Mapped syntax.
Line 10: I'm not sure actually, I just followed the syntax from the hello-book-api.

File: app/routes/board_route.py (hunk 3 above)
Comment (Msambere) [reply app/routes/board_route.py:13]: The create_model() function already has error handling inside. So this would just be doing it twice.

**Pull request merged**